src/leo-server-app/src/types/schedule.ts

```diff
@@ -1,4 +1,4 @@
-export enum ScheduleType {
-  "LIVE" = "LIVE",
```

Comment:
what happens to live execution

Reply:
We still add them to the queue, but we no longer need to differentiate between live and scheduled commands